src/main/java/seedu/moneygowhere/logic/commands/Command.java

```diff
@@ -17,4 +21,15 @@
      */
     public abstract CommandResult execute(Model model) throws CommandException;
 
+    public Map<Date, Double> getGraphData(Model model) {
+        return null;
+    }
+
+    public Map<Tag, Double> getStatsData(Model model) {
+        return null;
+    }
+
+    public String getStatsMessage(Model model) {
+        return null;
+    }
```

Comment:
May be create test cases for this 3 methods so that it will increase code coverage?
can try like assertEquals(null, new Command.getGrapdData(new Model))

Reply:
Done!